src/ol_infrastructure/substructure/keycloak/__main__.py

```diff
@@ -118,15 +118,15 @@
 )
 
 # Create Dagster OIDC client
-keycloak_domain_name = keycloak_config.require("url")
+dagster_domain_name = keycloak_config.require("url")
```

Comment:
This would require a new config object scoped to the dagster prefix, or just a config.require with the full `dagster:url` key